openpype/plugins/publish/integrate.py

```diff
@@ -529,6 +529,15 @@ def prepare_representation(self, repre,
         template_data["representation"] = repre["name"]
         template_data["ext"] = repre["ext"]
 
+        # add template data for colorspaceData
+        if repre.get("colorspaceData"):
+            colorspace = repre["colorspaceData"]["colorspace"]
+            # replace spaces with underscores
+            # pipeline.colorspace.parse_colorspace_from_filepath
+            # is checking it with underscores too
+            colorspace = colorspace.replace(" ", "_")
+            template_data["colorspace"] = colorspace
```

Comment:
This seems a little hardcoded and not sure the reason for it?

Reply:
We are not supporting spaces in our path formatting. But If you check the code [here](https://github.com/ynput/OpenPype/pull/4195/files#diff-c4b595d393443040b1d7c9551351bfffe69f9c00ad87bd9b6243da47a643f337R163) were are supporting parsing from path on both variants, with or witout underscores. But the integreation will always convert spaces into underscores.